src/lib/elements/forms/inputText.svelte

```diff
@@ -62,9 +74,19 @@
             class:u-padding-inline-end-56={typeof maxlength === 'number'}
             bind:this={element}
             on:invalid={handleInvalid} />
-        {#if maxlength}
-            <TextCounter max={maxlength} count={value?.length ?? 0} />
-        {/if}
+        <ul
+            class="buttons-list u-cross-center u-gap-8 u-position-absolute u-inset-block-start-8 u-inset-block-end-8 u-inset-inline-end-12">
+            {#if maxlength}
+                <li class="buttons-list-item">
+                    <TextCounter max={maxlength} count={value?.length ?? 0} />
+                </li>
+            {/if}
+            {#if !required}
```

Comment:
We might not want to show the null button is some inputs that aren't required... I would show/hide it according to a `isNullable` variable that is exported :thinking:

Reply:
We discussed this in discord afterwards, can we leave it as is then?

Reply:
yes :)